modules/cert-manager-install-console.adoc

```diff
@@ -24,12 +24,23 @@ You can use the web console to install the {cert-manager-operator}.
 . Select the *{cert-manager-operator}* and click *Install*.
 
 . On the *Install Operator* page:
-.. The *Update channel* is set to *tech-preview*, which installs the latest Technology Preview release of the {cert-manager-operator}.
-.. The *Installation Mode* is set to *All namespaces on the cluster (default)*. This mode installs the Operator in the Operator-recommended `openshift-cert-manager-operator` namespace to watch and be made available to all namespaces in the cluster.
-.. Choose the *Installed Namespace* for the Operator. The default Operator recommended namespace is `openshift-cert-manager-operator`.
+.. The *Update channel* is set to *stable-v1*, which installs the latest stable release of the {cert-manager-operator}.
 +
-If the `openshift-cert-manager-operator` namespace does not exist, it is created for you.
-.. Click the *Enable Operator recommended cluster monitoring on the Namespace* checkbox to enable cluster monitoring for the Operator.
+// TODO: Right now I see "alpha", "stable-v1", "stable-v1.10", and tech-preview. Need to see if all of these will be available at GA or not, and if any shown should be avoided / descriptions provided
```

Comment:
@thejasn @swghosh Will all of these channels be available at GA? Is one of them preferred over the others or are there any that should be avoided?

Reply:
Recommended would be `stable-v1`

Reply:
Thanks!